bin/state

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-# shellcheck source=bin/_config.sh
-source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
-
 # Run Terraform commands on the create_state_bucket project in the Hashicorp
 # docker container, passing our environment variables
-DOCKER_USER="$(id -u):$(id -g)" \
-    docker-compose run --rm \
-        -v "${PWD}/src/aws/create_state_bucket:/app"\
-        terraform "$@"
+
+docker run --rm -it \
+    --env-file ./env.d/aws \
+    -u "$(id -u)" \
+    -v "${PWD}/aws/create_state_bucket:/app" \
+    -w "/app" \
+    "hashicorp/terraform" \
+    "$@"
```

Comment:
Why not using the docker-compose `terraform` service for this?

Reply:
I don't know I did not find an advantage to use docker-compose. The thing is I can't use the existing `terraform` service in docker-compose because if defines environment variables that I don't want here.

Reply:
Those variables are potentially harmful or without any effect for what you are trying to achieve?

Reply:
yes they should not be set when running bin/state